Sources_v3/StreamChatUI/Common Views/ChatSquareButton.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatSquareButton: Button {
+    open var defaultSize: CGFloat?
+    
+    public init(image: UIImage? = nil, tintColor: UIColor? = nil) {
+        super.init(frame: .zero)
+        setImage(image, for: .normal)
+        self.tintColor = tintColor
+    }
+    
+    override public func defaultAppearance() {
+        defaultSize = 20
```

Comment:
Are you sure about having a button only 20pt big? Isn't 44 the minimal recommended tapping area?

Reply:
@DominikBucher12 It will be possible for SDK users to override `setupAppearance()` and change this values so they will be applied in `setupLayout()` later.

@VojtaStavik Good point, I've changed it to `40` but there will be huge gaps if we will go with bigger numbers.